frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.jsx

```diff
@@ -247,7 +248,12 @@ export class TagEditorParam extends Component {
 
         {(hasWidgetOptions || !isDimension) && (
           <div className="pb4">
-            <h4 className="text-medium pb1">{t`Filter widget label`}</h4>
+            <h4 className="text-medium pb1">
+              {t`Filter widget label`}
+              {hasNoWidgetLabel && (
+                <span className="text-error mx1">{t`(required)`}</span>
```

Comment:
Ideally we shouldn't use global css classes in new code because it increases the scope of migration